.env.example

```
@@ -0,0 +1,13 @@
+# Database (Neon)
+DATABASE_URL=postgres://user:pass@ep-xxx.us-east-2.aws.neon.tech/monkeysplit?sslmode=require
+
+# Auth (Auth.js)
+NEXTAUTH_URL=http://localhost:3000
+NEXTAUTH_SECRET=  # Generate: openssl rand -base64 32
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)